XSConsoleConstants.py

```diff
@@ -0,0 +1,2 @@
+# Default NTP domains are <int>.[centos|xenserver].pool.ntp.org
+DEFAULT_NTP_DOMAINS = [".centos.pool.ntp.org", ".xenserver.pool.ntp.org"]
```

Comment:
Suggestion to move the static definition of the default NTP server domain to the constants:

```suggestion
CENTOS_NTP_DOMAIN = "centos.pool.ntp.org"
DEFAULT_NTP_DOMAINS = [CENTOS_NTP_DOMAIN, "xenserver.pool.ntp.org"]

DEFAULT_NTP_SERVERS = ["%d.%s" % (i, CENTOS_NTP_DOMAIN) for i in range(4)]
```